modules/sdk-core/src/bitgo/wallet/iWallet.ts

```diff
@@ -420,6 +420,7 @@ export interface SubmitTransactionOptions {
     txHex?: string; // Transaction in any format required by each coin, i.e. in Tron it is a stringifyed JSON
     payload?: string;
     txBase64?: string;
+    signedChildPsbt?: string; // PSBT that has the output of txHex as input with a signature
```

Comment:
This seems to be a coin specific option? is it necessary to add like this here?. Also can we get a additional review from someone from your team. Thanks

Reply:
we are currently making changes in wp's `/send` to accept `signedChildPsbt` in `halfSigned` We did this way because there is no such thing as `coinSpecific` in `/send`'s body parameter. Here is the pr that makes these changes in wp: https://github.com/BitGo/bitgo-microservices/pull/29459

Reply:
Added @brandonblack as an additional review 👍

Reply:
Okay, Then I would recommend to use some thing like `coinSpecific?: Record<string, unknown>;` so that it becomes more generic. You can checking this property `signedChildPsbt` to be present in `coinSpecific` in the API side for UTXO specific coins to do the custom logic for it. In this way, other coins in the future can also utilise this.

Reply:
Okay, Then I would recommend to use some thing like `coinSpecific?: Record<string, unknown>;` so that it becomes more generic. You can checking this property `signedChildPsbt` to be present in `coinSpecific` in the API side for UTXO specific coins to do the custom logic for it. In this way, other coins in the future can also utilise this.